static/translations/messages_en.json

```diff
@@ -118,7 +118,10 @@
   "Type a message": "Type a message",
   "Message group": "Message group",
   "starred": "starred",
-  "Enable notifications": "Enable notifications",
+  "Failed to subscribe": "Failed to subscribe",
+  "Failed to unsubscribe": "Failed to unsubscribe",
+  "Unpin from top": "Unpin from top",
+  "Failed to unpin from top": "Failed to unpin from top",
```

Comment:
In this and the next couple of commits, it looks like the changes in this file got a bit messed up.

I think the net changes at the end of the branch look right, though.